packages/terra-dropdown-button/src/SplitButton.jsx

```diff
@@ -49,6 +49,17 @@ const propTypes = {
    * The intl object to be injected for translations.
```

Comment:
https://github.com/cerner/terra-core/pull/2766#discussion_r360082208 needs to be addressed in this file too.

Reply:
I missed it, updated [here](https://github.com/cerner/terra-core/blob/3d4cbdca1a5c96ae211cdcf5bdb5847ecfdc965c/packages/terra-dropdown-button/src/SplitButton.jsx#L126-L127)